gini/gini-vision-lib-android
# Feature mobile 689 remove 4 3 requirement

## Dropped 4:3 requirement and limitation
We don't check, if the camera has a picture size larger than ~8MP with a 4:3 aspect ratio. We only check that there is a size larger than ~8MP.

The camera uses the largest resolution and the preview resolution is selected to match the picture size's aspect ratio.

## How to test
Run the Screen API or the Component API example app. The Samsung Galaxy S6 has a 16:9 largest resolution and you should see bars on the left and right in the Camera Screen.

Run the tests in `ginivision/src/androidTest` and `ginivision/src/test`.

[MOBILE-689](https://tickets.i.gini.net/issue/MOBILE-689)
